build.sbt

```diff
@@ -1376,6 +1373,9 @@ lazy val `jna-wrapper` = project
       Map(
         javaModuleName.value -> jna
       )
+    },
+    assemblyMergeStrategy := { case _ =>
+      MergeStrategy.preferProject
```

Comment:
Makes sure we get our [copy of Native without `java.awt` references](https://github.com/enso-org/enso/pull/13021/files#diff-cc3073d7b0e23e915f86b0cb847a90bdd005d78d72d043a5d7cf732ddf8d43f7R26) rather than original version from the dependency.